geoalchemy2/elements.py

```diff
@@ -253,6 +253,8 @@ class CompositeElement(FunctionElement):
     Instances of this class wrap a Postgres composite type.
     """
 
+    inherit_cache = True
+
```

Comment:
if I have two CompositeElement objects with a different "field" and "type_", is the identical SQL produced for the two objects?  is the return type of the SQL function exactly the same for both objects? if no, this has to be inherit_cache=False unless a cache key that takes these fields into account is implemented (which as noted, I haven't made that API very public yet).

Reply:
Ok, I see, let's set it to False then.